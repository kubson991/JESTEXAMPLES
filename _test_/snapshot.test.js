import { getCharacter } from "../snapshot";
import rick from '../rick.json'
describe('es hora de las instantaneas', () => {
    test('Snapshot', () => {
        expect(getCharacter(rick)).toMatchSnapshot()       
    });    
    test('siempre fallara la instanea', () => {
        const user={
            createAt:new Date(),
            id:Math.floor(Math.random()*20)
        }   
        expect(user).toMatchSnapshot() 
    });
    test('tenemos una execpcion', () => {
        const user={
            id:Math.floor(Math.random()*20) ,
            name:"Oscar Barajas"   
        }
        expect(user).toMatchSnapshot({
            id: expect.any(Number)
        })
    });
});