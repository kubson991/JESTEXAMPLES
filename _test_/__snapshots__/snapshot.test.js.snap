// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`es hora de las instantaneas Snapshot 1`] = `
Object {
  "gender": "Male",
  "id": 1,
  "name": "Rick Sanches",
  "species": "Human",
  "status": "Alive",
}
`;

exports[`es hora de las instantaneas siempre fallara la instanea 1`] = `
Object {
  "createAt": 2021-08-09T23:09:49.524Z,
  "id": 15,
}
`;

exports[`es hora de las instantaneas tenemos una execpcion 1`] = `
Object {
  "id": Any<Number>,
  "name": "Oscar Barajas",
}
`;
